nilearn/stats/second_level_model/second_level_model.py

```diff
@@ -242,6 +242,14 @@ class SecondLevelModel(BaseEstimator, TransformerMixin, CacheMixin):
         parameters. Automatic mask computation assumes first level imgs have
         already been masked.
 
+    target_affine : 3x3 or 4x4 matrix, optional
+        This parameter is passed to nilearn.image.resample_img.
```

Comment:
```suggestion
        This parameter is passed to :func:`nilearn.image.resample_img`.
```
(to have the linking on the online documentation)